include/alpaka/warp/WarpSingleThread.hpp

```diff
@@ -92,6 +92,22 @@ namespace alpaka
                     return predicate ? 1u : 0u;
                 }
             };
+
+            //#################################################################
+            template<>
+            struct Shfl<WarpSingleThread>
+            {
+                //-------------------------------------------------------------
+                static auto shfl(warp::WarpSingleThread const& /*warp*/, int val, int /*srcLane*/)
```

Comment:
I have seen unused argments in alpaka usually named and cast to `void` inside the function body. I guess the argument was better documentation.

Reply:
I think we only mandatory do it when the arguments are conditionally used (e.g. inside an `#ifdef` guard), to avoid a warning when the condition is not met. This one is probably fine.